tests/install_swiftpm_dependencies/workspace_and_project/Makefile

```
@@ -0,0 +1,2 @@
+generate:
+	xcodegen generate --spec ../project/project.yml
```

Uh, interesting, I guess doing so broke it. I'm guessing this is the reason?
```suggestion
	# Note: we need to copy the project file in place (as opposed to using --spec)
	# so that xcodegen can add the scheme for the Demo project into the `Demo.xcworkspace`
	cp ../project/project.yml .
	xcodegen generate
```

---

I should have documented it myself. Thanks! 👍